control-plane-pki-operator/operator.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	configv1client "github.com/openshift/client-go/config/clientset/versioned"
+	configv1informers "github.com/openshift/client-go/config/informers/externalversions"
+	hypershiftv1beta1 "github.com/openshift/hypershift/client/clientset/clientset/typed/hypershift/v1beta1"
+	"github.com/openshift/hypershift/control-plane-pki-operator/certrotationcontroller"
+	"github.com/openshift/hypershift/control-plane-pki-operator/targetconfigcontroller"
+	"github.com/openshift/library-go/pkg/controller/controllercmd"
+	"github.com/openshift/library-go/pkg/operator/certrotation"
+	"github.com/openshift/library-go/pkg/operator/configobserver/featuregates"
+	"github.com/openshift/library-go/pkg/operator/status"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+)
+
+func RunOperator(ctx context.Context, controllerContext *controllercmd.ControllerContext) error {
+	var namespace, name string
```

Comment:
@deads2k would like your guidance on parameterizing this - the controllercmd libarry does not allow for flags and will require hackery one way or ther other ...

Reply:
> @deads2k would like your guidance on parameterizing this - the controllercmd libarry does not allow for flags and will require hackery one way or ther other ...

we own the library.  Adjust as needed to do what you want.

Reply:
Just used `os.Getenv`, there's a little too many layers of complexity in `controllercmd` for me to grok for now.